packages/nvidia_nat_core/src/nat/llm/dynamo_llm.py

```
@@ -564,14 +564,25 @@ async def handle_async_request(self, request: "httpx.Request") -> "httpx.Respons
                     osl_raw = int(prediction.output_tokens.p90)
                     iat_raw = int(prediction.interarrival_ms.mean)
 
+                    # Auto-assign latency sensitivity from profiler data
+                    # Only if prediction has it AND no manual @latency_sensitive decorator is active
+                    if prediction.latency_sensitivity is not None:
+                        try:
+                            ctx = Context.get()
+                            if not ctx.has_manual_latency_sensitivity:
+                                latency_sensitivity = prediction.latency_sensitivity
+                        except Exception:
+                            pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)